datamodel/src/rendering/Shader.cpp

```diff
@@ -0,0 +1,124 @@
+#include <fstream>
+
+#include <rcbe-engine/datamodel/rendering/Shader.hpp>
+
+#include <rcbe-engine/core/gl_extensions.hpp>
+
+#include <boost/log/trivial.hpp>
+
+namespace rcbe::rendering {
+int Shader::MAXIMUM_VERTEX_ATTRIBUTES = 0;
+
+Shader::~Shader() {
+    glDeleteShader(shader_handle_);
+}
+
+Shader::Shader(const core::EnginePath &path, ShaderType type)
+:
+type_(type)
+, file_path_(path)
+{
+    if (!std::filesystem::exists(file_path_)) {
```

Comment:
If you added `EnginePath` then use it.

Reply:
This is not a type but a static function :smile: